### PR TITLE
`TotalMembership` added in `CongressVotingResult`

### LINKED ISSUE #623: Add total membership field and PR hash to congress-voting-result
Currently voting condition is `number of yes - number of no should be more than 10% of total membership count`.
For this condition is reflected to congress-voting-result operation,
we need total membership field in congress-voting-result operation.
And congress-voting-result operation should refer saved proposal.
We need also Proposal hash field in congress-voting-result operation.

### PR DESCRIPTION
**Pull request by spikeekips**
### Github Issue
Resolves #623 

### Background

See #623 

### Solution

* As title said, `TotalMembership` added in `CongressVotingResult`
* test case added.

### CODE REVIEW

File: lib/client/response.go [hunk 1]
```diff
@@ -125,6 +125,7 @@ type CongressVotingResult struct {
 		No    uint64 `json:"no"`
 		ABS   uint64 `json:"abs"`
 	} `json:"result"`
+	TotalMembership uint64
```
Comment (soonkuk) [inline lib/client/response.go]: @spikeekips I discussed this with @outsideris.
@outsideris thought that to verify membership count from user-side, each membership's hash and membership.URL should be also included like voting result.

Comment (spikeekips) [reply lib/client/response.go]: @soonkuk Sorry, I misunderstood, I will make it again :)

**Comment by outsideris** [comment]
I didn't fully test voting result yet. I want to try to this feature with our voting server. 
Can I provide feedback after integration? Or I need open another issue after I reviewed this PR?

**Comment by spikeekips** [comment]
> @outsideris : I didn't fully test voting result yet. I want to try to this feature with our voting server.
Can I provide feedback after integration? Or I need open another issue after I reviewed this PR?

Your testing result will be important, I think. Fee free to test, I can wait :)

**Comment by spikeekips** [comment]
@outsideris Please complete to review :)

**Comment by spikeekips** [comment]
@outsideris Please complete review :)

**Comment by spikeekips** [comment]
@outsideris Is your test finished? Please let me know. This PR has been stayed too long in list. :(

**Comment by outsideris** [comment]
@spikeekips Not yet. Sorry. I'm working on voting in desktop app. I hope I can implement voting result tomorrow. And please rebase them.

**Comment by outsideris** [comment]
Please rebase them onto master. I want to test it locally.

**Comment by spikeekips** [comment]
@outsideris rebased.

**Pull request merged**